src/Components/GEOSldas_GridComp/GEOSlandpert_GridComp/GEOS_LandPertGridComp.F90

```diff
@@ -2448,7 +2449,7 @@ subroutine ApplyPrognPert(gc, import, export, clock, rc)
             ! Weiyuan notes: propagate_pert is called in GenerateRaw, not here
             diagnose_pert_only=.true.                                           &
             )
-       call MAPL_TimerOn(MAPL, '-GetPert')
+       call MAPL_TimerOff(MAPL, '-GetPert')
```

Comment:
@weiyuan-jiang : I think I misunderstood our conversation on Teams.  I had thought the "timer fix" had to do with an interface change imposed by MAPL.   In looking at the src code change here, this seems to just be a plain old bug fix that is needed no matter which MAPL version (2.10.1 or 2.14.0) is used, right?

Reply:
Right. The new MAPL exposes this bug and it must be fixed